docs/tour/adding-capabilities.mdx

```diff
@@ -1,30 +1,34 @@
 ---
-title: 'Add Capabilities'
+title: 'Plug in Capabilities'
```

Comment:
```suggestion
title: 'Add Features'
```

What we really want developers to do with wasmCloud is choose a language and get right down to developing features for applications, aka business logic, and then ship em. The second thing we want people to do immediately after the install + hello world is to write impactful code and see it work. Now we need this to be done with an interface driven model, and interact with capabilities, but I think we can still handle that here without them really _knowing_ that's what's going on.

I think you nail this in the first section. Identify your applications needs, bring in the interface, then write the code. This page should be about writing the code, because `wash dev` can hook up all the wiring.
<img width="875" alt="image" src="https://github.com/user-attachments/assets/a09e859b-8382-437c-b557-07d4dd796d73">

All that being said, what do you think about this title? Or maybe "write code" or "develop an app"? 🤔 

I think based on this philosophy we could make a few wording changes to the page

Reply:
What I want to make clear is that wasmCloud sees that the user imports the interface, and automatically satisfies the capability requirement. Maybe we mention that it uses NATS KV, maybe not, but the hook to move to the next page should be the knowledge that it's all automagical but we can plug in our own